readme.md

```diff
@@ -2,43 +2,40 @@
 
 > [Kap](https://github.com/wulkano/kap) plugin - Upload recordings to [Dropbox](https://dropbox.com)
 
-
 ## Install
 
-In the `Kap` menu, go to `Preferences…`, select the `Plugins` pane, find this plugin, and click `Install`.
-
+In the `Kap` menu, go to `Preferences…`, select the `Plugins` pane, find this plugin, and toggle it.
 
 ## Usage
 
 In the editor, after recording, select a format, and then `Upload to Dropbox`.
 
 ## Options
 
-In the `plugins` pane, click `Open plugins folder` and edit the `kap-dropbox.json` file to change options
+In the “Plugins” pane, click the pencil icon to edit the plugin options.
 
-Available options:
+Alternatively, you can manually edit the options in your text editor; Click `Open plugins folder` and edit the `kap-dropbox.json` file to change options.
 
 #### path
-Type: `String`
 
-When provided, uploads will be placed in that directory. For example, to upload recordings under a directory called Kaptures, set `path` to `/Kaptures`. If the directory doesn't exist it will be created.
+Type: `string`
+
+When provided, uploads will be placed in that directory. For example, to upload recordings under a directory called `Kaptures`, set `path` to `/Kaptures`. If the directory doesn't exist it will be created.
```

Comment:
Does it really have to be `/Kaptures` with a `/` prefix?

Reply:
It's the way the API expects it, so I kept it the same